fury/ui/containers.py

```diff
@@ -619,6 +619,7 @@ def __init__(
         inactive_color=(0.5, 0.5, 0.5),
         draggable=False,
         startup_tab_id=None,
+        tabBar_pos="top",
```

Comment:
Please follow `snake_case` while naming the variable.
Something like `tab_bar_pos` `tab_header_pos` or `tab_panel_pos`